.ci/java-versions.properties

```diff
@@ -7,5 +7,6 @@
 ES_BUILD_JAVA=openjdk12
 ES_RUNTIME_JAVA=java11
 GRADLE_TASK=build
+GRADLE_EXTRA_ARGS=-Dtests.bwc.refspec.7.x=peer-recovery-retention-leases-7.x
```

Comment:
There's a risk that we forget to remove this when we merge these branches to master. The consequence will be that `master` builds start to use `peer-recovery-retention-leases-7.x` rather than `7.x`, which will fail when:

- we delete the `peer-recovery-retention-leases-7.x`, or
- we introduce a BWC-affecting change to `master` and `7.x` (but, of course not to `peer-recovery-retention-leases-7.x`)

I think we'll catch this mistake pretty quickly, but if it's a concern then ideas for machinery to fail the build sooner are welcome.

Reply:
it's fine